pkg/scheduler/schedule_one.go

```diff
@@ -397,7 +397,7 @@ func (sched *Scheduler) findNodesThatFitPod(ctx context.Context, fwk framework.F
 	}
 	// Run "prefilter" plugins.
 	preRes, s := fwk.RunPreFilterPlugins(ctx, state, pod)
-	if !s.IsSuccess() {
+	if !s.Passed() {
```

Comment:
Let's imagine an scenario where all the plugins are skipped. Wouldn't the framework runtime return Success?

So this change shouldn't matter, right?

Reply:
I have one thing need to apologize to you related to this; I realized I had misunderstood and the bug (which was the reason I reverted the previous PR) didn’t actually exist. so I should not have reverted the previous PR. I’m so sorry about this and to take up your extra time to review this PR. 🙇 🙇 

So, you are correct at this point, actually, this change doesn’t matter. Also, we probably don’t need to create `Passed()` function because any upstream function should not receive Skip.
Let me revert those useless changes.

Reply:
Can you add a test case to ensure that RunPreFilterPlugins to return Success when all plugins return Skip?

Reply:
No need to apologize. Better be safe than sorry :)

Reply:
> Can you add a test case to ensure that RunPreFilterPlugins to return Success when all plugins return Skip?

We have already had `"all PreFilter plugins returned skip",` test case in pkg/scheduler/framework/runtime/framework_test.go#L1475.